packages/components/src/PieChart/PieChart.component.js

```diff
@@ -37,6 +37,62 @@ export function getAngle(percentage) {
 	return ((percentage * 2) / 100) * Math.PI;
 }
 
+/**
+ * This function return the showed value on the chart
+ * @param {array} model the pie chart model
+ * @param {index} index current index showed
+ */
+function getShowedValue(model, index) {
+	if (!model) {
+		return {};
+	}
+	return model[index];
+}
+
+/**
+ * This function is just a sort
+ * @param {object} a an element with a percentageShown to compare
+ * @param {object} b an other element with a percentageShown to compare
+ */
+function sortElements(a, b) {
```

Comment:
Although it's legacy, this could be simplified to `(a, b) => b.percentageShown - a.percentageShown)`